src/Tribe/Customizer/Global_Elements.php

```diff
@@ -121,59 +172,156 @@ public function get_css_template( $template ) {
 				}
 			';
 
+			// overrides for tec components/full/_events-bar.pcss
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 8f97ca11c5457e9328982ed8ab39c5cd</sub>